bool[] left_eye
bool[] right_eye
int32 eye_width
int32 eye_height
string eye_name

bool[] mouth
int32 mouth_width
int32 mouth_height
string mouth_name
string mouth_description